pod/perldelta.pod

```diff
@@ -27,6 +27,23 @@ here, but most should go in the L</Performance Enhancements> section.
 
 [ List each enhancement as a =head2 entry ]
 
+=head2 Subroutine signatures are no longer experimental
+
+Introduced in Perl version 5.20.0, and modified several times since, the
+subroutine signatures feature (C<use feature 'signatures'>) is now no longer
+considered experimental. It is now considered a stable language feature and
+is included in the C<:5.36> feature bundle, enabled automatically by
+C<use v5.36>, and no longer prints a warning.
+
+    use v5.36;
+
+    sub add ($x, $y) {
+      return $x + $y;
+    }
+
+Despite this, certain elements of signatured subroutines remain
+experimental; see below.
+
 =head2 @_ is now experimental within signatured subs
 
 Even though subroutine signatures currently remain experimental, use of the
```

Comment:
I think the description of the section about `@_` in signatured subs also needs to be adjusted.

Reply:
Ahyes. Further rewordings force-pushed.